### ISSUE TITLE
Providing more experiment details to achieve the paper scores.

### ISSUE THREAD

**Issue by zokost** [Leiay/looped_transformer#1]
Hello, you made a great research on the topic! Is it possible to share one of your best configs? I try to train your pipeline and get poor results even when I use a base config and try to take the parameters from the paper (I'm training on linear regression task)
![image](https://github.com/user-attachments/assets/d9e548ca-937a-4b32-9c09-e16cdd0f0e6e)

**Comment by Leiay** [comment]
Hi @zokost , thanks for your interest to our work! The config provided in the [base.yaml](https://github.com/Leiay/looped_transformer/blob/main/configs/base.yaml) should be enough to reproduce the vanilla transformer's performance. This figure looks strange to me, as it seems the model does not learn anything. Could you please share more details, for instance, the wandb training log? Does the training loss decrease at all?

**Comment by zokost** [comment]
I'm sorry, there was my mistake with model saving. Thank you for the quick response!

**Comment by Leiay** [comment]
Thank you for your reply. I'm glad you found the issue!
